fftools/ffmpeg_opt.c

```diff
@@ -569,6 +566,49 @@ static int opt_vaapi_device(void *optctx, const char *opt, const char *arg)
 }
 #endif
 
+#if CONFIG_QSV
+static int opt_qsv_device(void *optctx, const char *opt, const char *arg)
+{
+#if CONFIG_VAAPI
+    const char *prefix = "vaapi=__qsv_child_device:";
+#elif CONFIG_DXVA2
+    const char *prefix = "dxva2=__qsv_child_device:";
```

Comment:
why it has __?

Reply:
__qsv_child_device is the name for a hw device. Using __ as prefix to reduce the possibility of using the same device name with user's device. 

I also added a warning of not use __qsv_child_device or __qsv_device as device name in user's commandline.